packages/dso-toolkit/src/components/accordion/accordion.args.ts

```diff
@@ -33,20 +33,18 @@ export interface AccordionArgs {
   labelStatus: LabelStatus;
 }
 
-export const accordionArgs: Pick<AccordionArgs, "demoScrollIntoView" | "open" | "handleTitle"> = {
+export const accordionArgs: Pick<AccordionArgs, "demoScrollIntoView" | "open" | "handleTitle" | "variant"> = {
   open: false,
   demoScrollIntoView: undefined,
   handleTitle: "ongewijzigd",
+  variant: "default",
 };
 
 export const accordionArgTypes: ArgTypes<AccordionArgs> = {
   variant: {
-    options: [undefined, "compact", "conclusion", "neutral", "compact-black"],
+    options: ["default", "compact", "conclusion", "neutral", "compact-black"],
```

Comment:
Net zoals ik hierboven verwacht ik

```suggestion
    options: [undefined, "default", "compact", "conclusion", "neutral", "compact-black"],
```